.github/workflows/trunk-upgrade.yaml

```diff
@@ -0,0 +1,34 @@
+name: Trunk Upgrade
+
+on:
+  schedule:
+    # Every Monday @ 8am
```

Comment:
Since we'll be receiving these upgrade PRs across a bunch of different repos... let's turn down the cadence on this. I'm thinking we turn it down to once a month. Mind updating?

Reply:
This is awesome btw 💯 😄

Reply:
Makes sense 👍  Done.